app/src/main/java/org/sopt/collaboration/campuspick/domain/model/BottomSheetSelect.kt

```diff
@@ -0,0 +1,53 @@
+package org.sopt.collaboration.campuspick.domain.model
+
+enum class DeadLine(val label: String) {
+    EMPTY(""),
+    ALL("전체"),
+    UNDER_WEEK("7일 이하"),
+    OVER_WEEK("7일 이상"),
+    OVER_TWO_WEEK("14일 이상"),
+    OVER_MONTH("30일 이상");
+
+    companion object {
+        fun DeadLine.Companion.fromLabel(label: String): DeadLine? {
+            return entries.find { it.label == label }
+        }
```

Comment:
`EMPTY`를 아무것도  선택되지 않은 상태로 의도하신 게 맞을까요?
그런 거라면
```suggestion
        fun DeadLine.Companion.fromLabel(label: String): DeadLine {
            return entries.find { it.label == label } ?: EMPTY
        }
```
식으로 find를 못했을 때 Default를 반환할 수 있을 거 같은데, 반환을 nullable하게 하신 이유가 따로 있을까요?

Reply:
오 enum장인 역시
저는 EMPTY는 단순 뷰모델에서 빈 값을 나타내기 위해 사용하고, ui에서는 null을 반환해서 렌더링되지 않게 하려고 했는데
이렇게도 사용가능하네요

Reply:
지금은 null 예외처리 할 경우가 없으니 non null로 만드는게 좋아보입니다.